### PR TITLE
EDU-167 create room data model

### PR DESCRIPTION
**Pull request by laurentiu-ilici**

### CODE REVIEW

File: migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_12_20_01_create_rooms_colletion {
```
Comment (ahelmberger) [inline migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js:2]: Typo collection

File: migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    await this.db.createCollection('rooms');
+    await this.db.createCollection('roomInvitations');
+
+    await this.db.collection('rooms').createIndexes([
+      {
+        name: '_idx_owner_',
+        key: { owner: 1 }
+      },
+      {
+        name: '_idx_members_user_id',
```
Comment (ahelmberger) [inline migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js:17]: For consistency we need another underscore at the end

File: migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js (hunk 1 above)
Comment (laurentiu-ilici) [reply migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js]: Fixed, thanks

File: migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js (hunk 2 above)
Comment (laurentiu-ilici) [reply migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js]: Fixed in both files. Thanks!

**Comment by ahelmberger** [comment]
There is room-service, but room**s**-controller, room**s**-store and room-invitation**s**-store. I think that all services, stores and controllers so far use the singular, while the collection specs (and the collection names) use the plural, maybe we should keep that consistent.

**Comment by laurentiu-ilici** [comment]
> There is room-service, but room**s**-controller, room**s**-store and room-invitation**s**-store. I think that all services, stores and controllers so far use the singular, while the collection specs (and the collection names) use the plural, maybe we should keep that consistent.

Fixed

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**